test/runtests.jl

```diff
@@ -42,7 +42,7 @@ Copy test sqlite file to temp directory, path `test.sqlite`,
 overwrite file if necessary and set permissions.
 """
 function setup_clean_test_db(f::Function, args...)
-    dbfile = joinpath(dirname(pathof(SQLite)), "../test/Chinook_Sqlite.sqlite")
+    dbfile = joinpath(@__DIR__, "Chinook_Sqlite.sqlite")
```

Comment:
can we revert this change? I like the `pathof` style since it makes it much easier for running this locally, step-wise, in the REPL.